controller/uninstall_controller.go

```diff
@@ -617,6 +622,23 @@ func (c *UninstallController) deleteReplicas(replicas map[string]*longhorn.Repli
 	return
 }
 
+// deleteLeftBackups deletes the backup having no backup volume
+func (c *UninstallController) deleteLeftBackups(backup *longhorn.Backup) (err error) {
+	backupVolumeName, ok := backup.Labels[types.LonghornLabelBackupVolume]
+	if !ok {
+		// directly delete it if there is even no backup volume label
+		c.ds.DeleteBackup(backup.Name)
+	}
+	_, err = c.ds.GetBackupVolume(backupVolumeName)
+	if err != nil && apierrors.IsNotFound(err) {
+		if err = c.ds.DeleteBackup(backup.Name); err != nil {
+			return errors.Wrap(err, "failed to mark for deletion")
+		}
+		return nil
```

Comment:
no need to error out when running into a `NotFound` error when deleting a backup.

Reply:
Fixed, thanks